React/CoreModules/RCTAppearance.mm

```diff
@@ -128,7 +128,9 @@ - (instancetype)init
 RCT_EXPORT_SYNCHRONOUS_TYPED_METHOD(NSString *, getColorScheme)
 {
 #if !TARGET_OS_OSX // [TODO(macOS GH#774)
-  _currentColorScheme = RCTColorSchemePreference(nil);
+  if (_currentColorScheme == nil) {
+    _currentColorScheme = RCTColorSchemePreference(nil);
```

Comment:
It is, but macOS doesn't do lazy initialization because macOS forces us to fetch the color scheme preference from the main thread, as seen here:

https://github.com/microsoft/react-native-macos/blob/5d84d16100b8eeeb33c075ffc5f486347393a75c/React/CoreModules/RCTAppearance.mm#L118-L126